src/parsers/manifest/dash/indexes/template.ts

```diff
@@ -149,8 +149,8 @@ export default class TemplateRepresentationIndex implements IRepresentationIndex
                     timescale,
                     indexRange: index.indexRange,
                     indexTimeOffset,
-                    initialization: index.initialization == null ?
-                      undefined :
+                    initialization: index.initialization === undefined ?
+                      null :
```

Comment:
I don't get how it worked before with `undefined`

Reply:
Actually, if the `initialization` was undefined :
- `range` was `undefined`
- `mediaURL` was `null` : In the pipelines, we considered trying to load this init segment. In segment loader, if the `mediaURL` is `null`, just send back a data-created event. It means that this case only concerns smooth segments being created by RxPlayer. So the smooth index still returns a init segment with `mediaURL === null`, while the other indexes just don't return an init segment, which seems normal.